invokeai/frontend/web/src/features/lora/components/ParamLora.tsx

```diff
@@ -22,7 +22,7 @@ const ParamLora = (props: Props) => {
   );
 
   const handleReset = useCallback(() => {
-    dispatch(loraWeightChanged({ id: lora.id, weight: 1 }));
+    dispatch(loraWeightChanged({ id: lora.id, weight: 0.75 }));
```

Comment:
I suggest defaulting LoRA weight to 0.75, as 1.0 is almost always too strong and we use 0.75 as the default in the 2.3 branch

Reply:
That would be ideal. When I used to use Recoil, I could just called a reset function that would just reset the state to its default. I am not sure if Redux has something like that built in. So we can avoid making a tun of reset actions.